src/main/java/com/suse/salt/netapi/calls/LocalCall.java

```diff
@@ -83,6 +102,9 @@ public LocalCall(String functionName, Optional<List<?>> arg,
         arg.ifPresent(arg -> payload.put("arg", arg));
         kwarg.ifPresent(kwarg -> payload.put("kwarg", kwarg));
         metadata.ifPresent(m -> payload.put("metadata", m));
+        timeout.ifPresent(timeout -> payload.put("timeout", timeout));
```

Comment:
I would say that the Batch case should be fixed in Salt, as ATM it is a Salt bug. That should not block this PR anyway, as the library is doing the  right thing FMPOV.

Is that OK with you @lucidd?